docs/containers.conf.5.md

```diff
@@ -405,7 +405,7 @@ if you want to set environment variables for the container.
 
 Define where event logs will be stored, when events_logger is "file".
 
-**events_logfile_max_size**=0
+**events_logfile_max_size**="1Mb"
 
 Sets the maximum size for events_logfile_path in bytes. When the limit is exceeded,
```

Comment:
The comment needs an update.  It's not in `bytes` anymore but in a human-readable form such as "1mb" or "42mb" etc.

Reply:
Same for the other comments in the code and config.